docs/configuration.md

```diff
@@ -214,6 +214,19 @@ services.bitcoind = {
 };
 ```
 
+> Note: If a `secure-node.nix` or `tor-enable.nix` preset is imported in your
+> configuration or a `tor.enforce` option is explicitly enabled, you also need
```

Comment:
Perhaps also mention that this only applies if the remote bitcoind isn't accessed via Tor?